Deployment take 2

orta opened the pull request:
Will comment inline

orta commented:
Merge'd master, should be good to go, last fail was danger unable to find a commit due to a force push.

ArtsyMobile commented:
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

orta commented:
:green_apple:

alloy commented on fastlane/Fastfile:
```diff
@@ -11,18 +12,79 @@ lane :ship_beta do
 end
+
+## We want to have a simpler version of Fastlane's match
+## this does it in a pretty reasonable amount of code.
```
Perfect :+1: Can we get rid of the `match` gem and its deps?

alloy commented on fastlane/Fastfile:
```diff
@@ -11,18 +12,79 @@ lane :ship_beta do
+  # Move our provisioning profile in
+  profile_path = File.expand_path("~") + "/Library/MobileDevice/Provisioning Profiles/"
+  destination = profile_path + "/" + app_name + ".mobileprovision"
+  profile = Dir.glob(signing_root + "/profiles/" + app_name + "/*").first
```
Better to use things like `File.join` then `+` etc, but no biggie.

orta commented:
OK, I've removed match, and fui - it wasn't being used. 

I've also updated Danger to this PR https://github.com/KrauseFx/danger/pull/40/ - which makes `--verbose` work - should make debugging DM easier for me.

alloy commented:
Awesome, can’t wait to deploy!
